### DISCUSSION TITLE
windows平台通过raidrive挂载后上传文件，20M以上速度直接为0

### DISCUSSION THREAD

**Discussion by cxh263590** [messense/aliyundrive-webdav#162] [Q&A]
通过pip 安装的aliyundrive-webdav, 并按照如下方式启动后，通过
aliyundrive-webdav --cache-size 100000 --refresh-token xxxxx  --read-buffer-size 1048576000
这里的xxxxx隐藏
![image](https://user-images.githubusercontent.com/25278882/142836814-eb4c2b4f-1470-42f0-9b38-3113efbed802.png)
然后raidrive 设置如上，挂载后，发现上传每到20M就传不动了，请问是什么原因呢

**Comment by messense** [comment]
`--read-buffer-size 1048576000` 开太大了？没有必要，默认值就差不多了。

最好能提供错误日志。

**Comment by cxh263590** [comment]
提示这个。 应该是上传有错误
![image](https://user-images.githubusercontent.com/25278882/143458817-f5bf9c01-7454-49a9-a09f-62efa325991d.png)
